### DISCUSSION TITLE
A few questions

### DISCUSSION THREAD

**Discussion by phil-pappas** [themefisher/airspace-hugo#170] [Q&A]
Hi everyone.

I have a few questions:
1. I tried to see if the email function works but I haven't receive anything in my email. Does it have to do that I run it locally?
2. How can I change the default language of the website? I want to change the English to Italian for example but I can't. It appears putting `defaultContentLanguage = "it"` in the config.toml file does not help
3. Is it possible to add new ionicon(or any other icon) except from the list here? (https://ionicons.com/v2/) If yes, then how?

Thanks in advance for your answers and all the hard work you put it.

**Comment by salim-b** [comment]
1. With "email function" you mean the contact form? If so, did you either enable Netlify Forms or set up a formspree account?
2. Putting `defaultContentLanguage = "it"` in the `config.toml` should work.
3. You would need to modify the theme for this. I guess a PR would be welcome since Ionicons v2 are quite outdated (I would go for [Bootstrap Icons](https://icons.getbootstrap.com/) these days).

**Comment by phil-pappas** [comment]
For the contact form:
I have `use_netlify = true` but I am running the theme locally. So I guess because I haven't deploy it yet to netlify, this is why it's happening? If yes, can't I use this function without deploying to netlify? (Except formspree option)

**Reply by salim-b** [reply]
> So I guess because I haven't deploy it yet to netlify, this is why it's happening?

Yes, Netlify needs to be able to parse the contact form (which it does during deploy). You can read more about it [here](https://docs.netlify.com/forms/setup/).

> If yes, can't I use this function without deploying to netlify? (Except formspree option)

No, I don't think so.
